modules/virt-adding-secret-configmap-service-account-to-vm.adoc

```diff
@@ -12,35 +12,42 @@ Add a secret, ConfigMap, or service account to a virtual machine by using the {p
 
 * The secret, ConfigMap, or service account that you want to add must exist in the same namespace as the target virtual machine.
 
-* The virtual machine must be powered off.
-
 .Procedure
 
-. From the side menu, click *Virtualization*.
+. Click *Workloads* -> *Virtualization* from the side menu.
 
-. Click the *Virtual Machine* tab.
+. Click the *Virtual Machines* tab.
 
-. Select a virtual machine to open its *Virtual Machine Overview* page.
+. Select a virtual machine to open the *Virtual Machine Overview* screen.
 
 . Click the *Environment* tab.
 
-. Click *Select a resource* and select a secret, ConfigMap, or service account from the list.
+. Click *Select a resource* and select a secret, ConfigMap, or service account from the list. A six character serial number is automatically
+generated for the selected resource.
 
 . Click *Save*.
 
 . Optional. Add another object by clicking *Add Config Map, Secret or Service Account*.
 
 [NOTE]
 ====
-You can reset the form to the last saved state by clicking *Reload*.
-====
+.. You can reset the form to the last saved state by clicking *Reload*.
 
+.. The *Environment* resources are added to the virtual machine as disks. You can mount the secret, ConfigMap, or service account as you would mount any other disk.
+
+.. If the virtual machine is running, changes will not take effect until you restart the virtual machine. The newly added resources are marked as pending changes
+for both the *Environment* and *Disks* tab in the *Pending Changes* banner on top of the page.
+
+====
 
 .Verification steps
 
 . From the *Virtual Machine Overview* page, click the *Disks* tab.
 
 . Check to ensure that the secret, ConfigMap, or service account is included in the list of disks.
 
-. Optional. Start the virtual machine by clicking *Actions* -> *Start Virtual Machine*.
+. Optional. If the virtual machine is running, restart the virtual machine by clicking *Actions* -> *Restart Virtual Machine*.
+
+. Optional. If the virtual machine is stopped, start the virtual machine by clicking *Actions* -> *Start Virtual Machine*.
```

Comment:
Because the user won't need to both start and restart the VM, I think these should be condensed into one step with two bullet points underneath it for the two different scenarios.